vending-app/src/main/kotlin/com/google/android/finsky/DeviceSyncInfo.kt

```diff
@@ -21,9 +21,9 @@ import android.text.TextUtils
 import android.util.Base64
 import android.util.DisplayMetrics
 import android.util.Log
+import android.os.Build
 import android.view.WindowManager
 import org.microg.gms.common.Constants
-import org.microg.gms.profile.Build
 import java.security.MessageDigest
```

Comment:
Loo at this: https://github.com/microg/GmsCore/commit/56b468ecbb450844c3e41a2b7d85ff979d98734b